script.js

```diff
@@ -0,0 +1,70 @@
+
+const searchButton = document.getElementById('searchButton');
+
+searchButton.addEventListener('click', () => {
+    document.getElementById("tableResults").innerHTML = '';
+    searchResults()
+    addSpinner();
+});
+
+async function searchResults() {
+    const symbol = document.getElementById('searchTerm').value;
+    const url = "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query="+ symbol + "&limit=10&exchange=NASDAQ"
```

Comment:
Instead of rewriting the API URL ("https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/) you should use (declare) baseUrl variable. Remember the DRY principle.